From 1022fb610af56ab834d756ec8fe0b0fd46aa11ab Mon Sep 17 00:00:00 2001
From: Andre Heider <a.heider@gmail.com>
Date: Wed, 23 Nov 2022 14:26:41 +0100
Subject: [PATCH 05/39] fix -Wmaybe-uninitialized warning

All previous cases not hit, which means no scan was possible.
Indicate that.

Signed-off-by: Andre Heider <a.heider@gmail.com>
---
 iwinfo_nl80211.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/iwinfo_nl80211.c b/iwinfo_nl80211.c
index 97cd39f..0e339b5 100644
--- a/iwinfo_nl80211.c
+++ b/iwinfo_nl80211.c
@@ -2899,6 +2899,8 @@ static int nl80211_get_scanlist(const char *ifname, char *buf, int *len)
 				iwinfo_ifup(ifname);
 				nl80211_hostapd_hup(ifname);
 			}
+			else
+				rv = -1;
 
 			nl80211_ifdel(res);
 			return rv;
-- 
2.35.1

